sessionize/lib/src/commonMain/kotlin/co/touchlab/sessionize/EventModel.kt

```diff
@@ -57,15 +57,15 @@ class EventModel(val sessionId: String) : BaseQueryModelView<Session, SessionInf
             if (reminderNotificationsEnabled()) {
                 ServiceRegistry.notificationsApi.createLocalNotification("Upcoming Event in " + event.session.room().name,
                         event.session.title + " is starting soon.",
-                        event.session.startsAt.toLongMillis() + AppContext.TEN_MINS_MILLIS,
+                        event.session.startsAt.toLongMillis() + Durations.TEN_MINS_MILLIS,
                         sessionId.hashCode(),
                         notificationReminderTag)
 
                 // Feedback Notifications
                 if (event.session.feedbackRating == null) {
                     val feedbackNotificationTime = event.session.endsAt.toLongMillis() + Durations.TEN_MINS_MILLIS
```

Comment:
would it be possible for this to just call createFeedbackNotificationsForSessions with a single session

Reply:
Unfortunately in this case it's a Session class whereas createFeedbackNotificationsForSessions takes in a MySession. We could make a shared function that just takes in simple types and use that.

Reply:
Possible Approach

`fun createFeedbackNotificationsForSessions<T>(sessions: List<T>) {
        if (feedbackEnabled() && notificationsEnabled()) {
            sessions.forEach { session ->

                (session as? MySessions)?.let{
                    if (session.feedbackRating == null) {
                        createFeedbackNotification(session.endsAt.toLongMillis(), session.id.hashCode())
                    }
                }
                (session as? Session)?.let{
                    if (session.feedbackRating == null) {
                        createFeedbackNotification(session.endsAt.toLongMillis(), session.id.hashCode())
                    }
                }
            }
        }
    }

    private fun createFeedbackNotification(endsAtTime: Long, sessionId: Int){
        val feedbackNotificationTime = endsAtTime + Durations.TEN_MINS_MILLIS
        ServiceRegistry.notificationsApi.createLocalNotification("Feedback Time!",
                "Your Feedback is Requested",
                feedbackNotificationTime,
                //Not great. Possible to clash, although super unlikely
                sessionId,
                notificationFeedbackTag)
    }`

Reply:
updated